pages/common/tailscale.md

```diff
@@ -0,0 +1,36 @@
+# tailscale
+
+> The easiest and most secure way to use WireGuard and 2FA.
```

Comment:
Shouldn't we keep the descriptions as objective as possible?

Reply:
The service describes itself with a lot of cool adjectives lol
```suggestion
> A private WireGuard network service.
```